CONTRIBUTING.md

```diff
@@ -0,0 +1,103 @@
+# Contributing to PaperMC Docs
+
+Thank you for considering contributing to the PaperMC Docs repository! We appreciate your help in making our documentation better. 
+To ensure a smooth and collaborative contribution process, please follow the guidelines below.
+
+## Table of Contents
+1. [Introduction](#introduction)
+2. [How to add a page](#adding-a-page)
+3. [Submitting Contributions](#submitting-contributions)
+4. [Style Guidelines](#style-guidelines)
+5. [Code of Conduct](#code-of-conduct)
+
+## Introduction
+
+The PaperMC Docs repository contains the documentation for the PaperMC project, which is used to provide essential 
+information, instructions, and guidelines to our users, developers, and contributors. Your contributions play a 
+significant role in improving and maintaining the quality of our documentation.
+
+We use [Docusaurus](https://docusaurus.io/) to build and deploy the documentation website. 
+Using Docusaurus means that there are guides for how to write the documentation. You can find the Docusaurus documentation
+[here](https://docusaurus.io/docs/category/guides).
+
+Our Docs are written in Markdown, which is a simple markup language that is easy to learn. 
+Take a look at some of the existing documentation to get a feel for how it is written.
```

Comment:
I would say, instead of "our docs", "The bulk of Paper's documentation is written with Markdown"